clients/summary/types.go

```diff
@@ -14,713 +14,285 @@ const (
 	SessionTokenScopes = "sessionToken.Scopes"
 )
 
-// BGMBucketData Series of counters which represent one hour of a users data
-type BGMBucketData struct {
-	LastRecordTime *time.Time `json:"lastRecordTime,omitempty"`
+// BgmperiodsV5 A map to each supported BGM summary period
+type BgmperiodsV5 map[string]GlucoseperiodV5
```

Comment:
These generated names make me sad. Is there no way to have them stick to std Go names for readability's sake? I.e. `BGMPeriodsV5` and `GlucosePeriodV5`?

Reply:
this is "fixed" in BACK-3631 / #75 , it requires a lot of file renames in TidepoolAPI.